traffic_ops/testing/api/v4/profile_parameters_test.go

```diff
@@ -76,6 +76,11 @@ func TestProfileParameters(t *testing.T) {
 					},
 					Expectations: utils.CkRequest(utils.HasError(), utils.HasStatus(http.StatusBadRequest)),
 				},
+				"BAD REQUEST when EMPTY BODY PROVIDED": {
+					ClientSession: TOSession,
+					RequestBody:   map[string]interface{}{},
+					Expectations:  utils.CkRequest(utils.HasError(), utils.HasStatus(http.StatusBadRequest)),
+				},
```

Comment:
This doesn't actually test the behavior being fixed in this PR. [Marshalling an empty map appropriately produces an empty object - not an empty request body](https://go.dev/play/p/A4w_-aQn2HB)!